README.md

```diff
@@ -2,886 +2,114 @@
-
-A Flow does not have to be local-only: You can put any Pod to remote(s). In the example below, with the `host` keyword `gpu-pod`, is put to a remote machine for parallelization, whereas other Pods stay local. Extra file dependencies that need to be uploaded are specified via the `upload_files` keyword.
-
-<table>
-    <tr>
-    <td>123.456.78.9</td>
-    <td>
-
-```bash
-# have docker installed
-docker run --name=jinad --network=host -v /var/run/docker.sock:/var/run/docker.sock jinaai/jina:latest-daemon --port-expose 8000
-# to stop it
-docker rm -f jinad
-```
-
-</td>
-</tr>
+  </tr>
   <tr>
-    <td>
-    Local
+    <th>
+      <a href="./.github/pages/hello-world.md#-fashion-image-search">Image search</a>
+    </th>
+    <th>
+      <a href="./.github/pages/hello-world.md#-covid-19-chatbot">Q+A search</a>
+    </th>
+    <th>
+      <a href="./.github/pages/hello-world.md#-multimodal-document-search">Multi-media search</a>
     </td>
-    <td>
-
-```python
-import numpy as np
-from jina import Flow
-
-f = (Flow()
-     .add()
-     .add(name='gpu_pod',
-          uses='mwu_encoder.yml',
-          host='123.456.78.9:8000',
-          parallel=2,
-          upload_files=['mwu_encoder.py'])
-     .add())
-
-with f:
-    f.index_ndarray(np.random.random([10, 100]), output=print)
-```
-</tr>
-
+  </tr>
 </table>
 
-We provide a demo server on `cloud.jina.ai:8000`, give the following snippet a try!
-
-```python
-from jina import Flow
+## What is Neural Search?
 
-with Flow().add().add(host='cloud.jina.ai:8000') as f:
-    f.index(['hello', 'world'])
-```
+Neural search is a new approach to retrieving information. Traditional symbolic search needs extensive rules and fragile pipelines. Neural search simply uses a pre-trained neural network to do the heavy lifting. This saves developers time and effort.
 
-#### Asynchronous Flow
-<a href="https://mybinder.org/v2/gh/jina-ai/jupyter-notebooks/main?filepath=basic-inter-intra-parallelism.ipynb"><img align="right" src="https://github.com/jina-ai/jina/blob/master/.github/badges/run-badge.svg?raw=true"/></a>
+With Jina you can search through any kind of data, whether that's [text](https://github.com/jina-ai/examples/tree/master/wikipedia-sentences), [images](https://github.com/jina-ai/examples/tree/master/pokedex-with-bit), [audio](https://github.com/jina-ai/examples/tree/master/audio-search) or [video](https://github.com/jina-ai/examples/tree/master/tumblr-gif-search). We leverage AI models that have pre-trained and released by Facebook, Google, Spotify, and more. With the modular framework you can find the right tool for the right job -- whether that's crafting, encoding, classifying, indexing, or querying data. Alternatively, assemble them all into one [Flow](https://101.jina.ai/#Flow) for a unified end-to-end experience. 
```

Comment:
I would remove simply, I think it make the sentence read more confidently 

I would also add "this approach", it's sounds better.

Reply:
Where would you add "this approach"?